### PR TITLE
[220] #CHANGE 'assemblyNmae: DotNet.Testcontainers; function OracleDb…

### PR DESCRIPTION
**Pull request by EvdAnton**
Change WaitStrategy for oracle testcontainers

### CODE REVIEW

**Comment by EvdAnton** [comment]
If you are going to add a solution with an oracle test container, tell me when you are going to add it to Nuget package?

**Comment by HofmeisterAn** [comment]
> If you are going to add a solution with an oracle test container, tell me when you are going to add it to Nuget package?

I can create a new package as soon as the PR is merged.

File: src/DotNet.Testcontainers.Tests/Fixtures/OracleDbFixture.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public class OracleDbFixture : IAsyncLifetime
     public OracleDbFixture()
```
Comment (HofmeisterAn) [inline src/DotNet.Testcontainers.Tests/Fixtures/OracleDbFixture.cs:17]: You need to rebase on top of `develop` first. I did some clean up. You had typos and inconsistent class / file names in your PR.

File: src/DotNet.Testcontainers/Containers/Configurations/Databases/OracleDbTestcontainerConfiguration.cs [hunk 2]
```diff
@@ -29,6 +29,12 @@ public override string Password
     }
 
     public override IWaitForContainerOS WaitStrategy => Wait.ForUnixContainer()
-      .UntilPortIsAvailable(this.DefaultPort);
+      .UntilCommandIsCompleted("bin/bash","-c",
+        "export ORACLE_SID=XE;"+
```
Comment (HofmeisterAn) [inline src/DotNet.Testcontainers/Containers/Configurations/Databases/OracleDbTestcontainerConfiguration.cs:33]: Are those exports really necessary?

**Review by HofmeisterAn — COMMENTED** [review]
Also check your commit message, there are errors too. Here is my suggestion:
```
[#220] #CHANGE 'assemblyName: DotNet.Testcontainers; function: OracleTestontainerConfiguration'
{Change WaitStrategy for Oracle testcontainer configuration}
```

**Comment by EvdAnton** [comment]
When i'm trying to fetch data from upstream/developer i have no changes in developer branch, that why i'm recreate repository and create new branch, pleas check again

**Pull request closed without merging**